packages/textfield/README.md

```diff
@@ -0,0 +1,267 @@
+# `<mwc-textfield>` & `<mwc-textarea>` [![Published on npm](https://img.shields.io/npm/v/@material/mwc-textfield.svg)](https://www.npmjs.com/package/@material/mwc-textfield)
+
+> IMPORTANT: The Material Web Components are a work in progress and subject to
+> major changes until 1.0 release.
+
+Textfields let users enter and edit text.
+
+<img src="images/action.png" width="28.5714%">
```

Comment:
Might avoid calendar icon examples, since we don't have a date picker yet.

Reply:
I used event to mimic material demos

https://material-components.github.io/material-components-web-catalog/#/component/text-field